.github/workflows/release.yml

```diff
@@ -88,4 +88,8 @@ jobs:
         secretDockerHubUser: ${{ secrets.TEST_DOCKER_HUB_USERNAME }}
         secretDockerHubPassword: ${{ secrets.TEST_DOCKER_HUB_PASSWORD }}
         secretGithubToken: ${{ secrets.GITHUB_TOKEN }}
+    - name: Deploy to ASF Snapshots Repository
+      working-directory: java
+      run: |
+        ./mvnw ${MAVEN_ARGS} clean deploy -DskipTests -DskipITs --settings .github/asf-deploy-settings.xml
```

Comment:
We may use `mvn` as there is no wrapper available in the java directory. I guess we need an action to install it though.

Reply:
Switched to `mvn`, seems like Maven it's already installed:
https://github.com/apache/camel-k/blob/f7554758925e530191db95b6f833286169d8b0ab/.github/workflows/release.yml#L85